root/static/scripts/guess-case/MB/GuessCase/Main.js

```diff
@@ -119,9 +119,10 @@ const flags = require('../../flags');
         sortname: guess("Work", "guessSortName")
     };
 
-    // Series doesn't have it's own handler, and just uses the work handler
-    // because additional behavior isn't needed.
+    // Series and Event dont have their own handler, and they use the
```

Comment:
Sorry for that silly mistake , I should have proofread it. I have made the changes